copis/gl/glcanvas.py

```diff
@@ -20,27 +20,35 @@
 
 import math
 import platform as pf
-from dataclasses import dataclass
-from threading import Lock
-from typing import List, NamedTuple
-
-import copis.gl.shaders as shaderlib
 import glm
 import numpy as np
 import pywavefront
 import wx
 
+from threading import Lock
+from typing import List, NamedTuple
+from wx import glcanvas
+from pydispatch import dispatcher
+from OpenGL.GL import ( shaders,
+    GL_ARRAY_BUFFER, GL_ELEMENT_ARRAY_BUFFER, GL_STATIC_DRAW,
+    GL_MULTISAMPLE, GL_FALSE, GL_UNSIGNED_BYTE, GL_FLOAT, GL_AMBIENT_AND_DIFFUSE,
+    GL_BLEND, GL_COLOR_BUFFER_BIT, GL_COLOR_MATERIAL, GL_CULL_FACE,
+    GL_DEPTH_BUFFER_BIT, GL_DEPTH_TEST, GL_FILL, GL_FRONT_AND_BACK, GL_LESS,
+    GL_LINE_SMOOTH, GL_ONE_MINUS_SRC_ALPHA, GL_RGB, GL_SRC_ALPHA, GL_TRIANGLES,
+    glGenVertexArrays, glUniformMatrix4fv, glDeleteBuffers, glGenBuffers,
+    glBindVertexArray, glEnableVertexAttribArray, glUseProgram,
+    glVertexAttribPointer, glBindBuffer, glBufferData, glBlendFunc, glClear,
+    glClearColor, glClearDepth, glColorMaterial, glDepthFunc, glDisable, glEnable, 
+    glPolygonMode, glViewport, glReadPixels, glDrawArrays)
+from OpenGL.GLU import ctypes
```

Comment:
Uhh _technically_ this is more Pythonic, but maybe for OpenGL stuff we can just wilcard import? Although I don't mind either way (I hope it won't get cumbersome to remember to import everything). Interested to hear your thoughts here.

Reply:
Although it was a pain to list them all, it allowed me to see exactly what all is used out of `OpenGL`. that in turn made it much easier to do targeted research on what each method does; instead of having to read the **ENTIRE** `OpenGL` API library documentation.